layers/meta-resin-intel-quark/recipes-core/initrdscripts/files/init-external-media.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+
+PATH=/sbin:/bin:/usr/sbin:/usr/bin
```

Comment:
Isn't this the same script used for nuc? Can we abstract this into a meta-resin feature? As well, why do we need the initramfs in normal images? Does it boot from usb or from the internal device?